### DISCUSSION TITLE
ap9440/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#14] [Ideas]
# ap9440/

本文最后更新于 2024-09-09，更新了后轴换用 Dixcel Z 刹车片后的成绩 对于刹车的改装，如我的 Nova Kwok 的思域 FK7 改车笔记 一文，是我觉得对于整车影响很大

https://nova.moe/ap9440/

**Comment by BlackTeaChan** [comment]
请问一下up，换了9440会和轮毂干涉吗？我的FK7装Model3轮毂+迈拓MT4会出现干涉无法安装，在更换了加长螺杆和垫片之后才不会打架

**Reply by n0vad3v** [reply]
我这个不会，不过可能和我选用的 CZV 套装有关（因为不同的合头和桥位可能厚度不一样）之前看到 Twitter 上有个车友也是使用 Model 3 轮毂，使用 Belfe 套装的 AP9440 就干涉了。
